docs/users/getting-started/configuration.md

```diff
@@ -65,15 +60,15 @@ Settings that affect all editors used inside BetterDiscord.
 
 ### Line Numbers
 
-This option simply determines whether or not line numbers should be shown.
+This option determines whether or not line numbers should be shown.
 
 ### Minimap
 
-This option determines if the minimap that represents the code on the right had side should be hidden or shown.
+This option determines if the minimap that represents the code on the right side should be hidden or shown.
 
 ### Reference Tooltips
 
-This option shows tooltips when hovering over parts of your code. For CSS this would be selector information. For JavaScript this would be variable information.
+This option shows tooltips when hovering over parts of your code. For CSS this would be selector information. For JavaScript, this would be variable information.
```

Comment:
```suggestion
This option shows tooltips when hovering over parts of your code. For CSS, this would be selector information. For JavaScript, this would be variable information.
```

If you're going to add the commas, be consistent! 😄